python/google/protobuf/json_format.py

```diff
@@ -399,9 +399,9 @@ def _CreateMessageFromTypeUrl(type_url, descriptor_pool):
   type_name = type_url.split('/')[-1]
   try:
     message_descriptor = pool.FindMessageTypeByName(type_name)
-  except KeyError:
+  except KeyError as e:
     raise TypeError(
-        'Can not find message descriptor by type_url: {0}'.format(type_url))
+        'Can not find message descriptor by type_url: {0}'.format(type_url)) from e
```

Comment:
This line is more than 80 columns side; please shrink it.  I suggest:

+   raise TypeError('Can not find message descriptor by type_url: {0}'.format(
+       type_url)) from e